_docs-sources/pipelines/what-is-it/index.md

```diff
@@ -1,8 +1,24 @@
 # Overview
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. At lectus urna duis convallis convallis tellus id interdum velit. Pretium vulputate sapien nec sagittis aliquam. Tristique senectus et netus et. Nullam eget felis eget nunc lobortis mattis aliquam faucibus purus. Aliquet sagittis id consectetur purus ut. Lectus magna fringilla urna porttitor rhoncus dolor. Eget felis eget nunc lobortis mattis aliquam faucibus purus in. Erat nam at lectus urna duis convallis convallis. Euismod lacinia at quis risus sed vulputate odio.
+Gruntwork Pipelines is a framework that enables you to use your preferred CI tool to
+securely run an end-to-end pipeline for infrastructure code (Terraform) and
+app code (Docker or Packer). Rather than replace your existing CI/CD provider, Gruntwork Pipelines is designed to enhance the security
+of your existing tool.
+
+Because applying arbitrary terraform requires full access to your AWS environment,
+your CI/CD tool would require admin level credentials to any AWS account where you deploy infrastructure.
+This makes it trivial for anyone with access to your CI/CD system (usually a long list) to access AWS credentials with permissions
```

Comment:
While it's potentially true that many people have access to a ci system, this implies that they are doing something wrong, i think we're better off without the comment

```suggestion
This makes it trivial for anyone with access to your CI/CD system to access AWS credentials with permissions
```